daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -48,11 +48,13 @@ data Command
       , openBrowser :: OpenBrowser
       , startNavigator :: Maybe StartNavigator
       , jsonApiCfg :: JsonApiConfig
+      , triggerServiceCfg :: TriggerServiceConfig
```

Comment:
I don’t think the trigger service is common enough to justify being part of `daml start`.

Reply:
Ok. Will unwind that bit.